hive-metastore/src/main/java/org/apache/iceberg/hive/HiveTableOperations.java

```diff
@@ -304,23 +310,13 @@ protected void doCommit(TableMetadata base, TableMetadata metadata) {
       throw new CommitFailedException(e);
 
     } finally {
-      cleanupMetadataAndUnlock(commitStatus, newMetadataLocation, lock);
+      HiveOperationsBase.cleanupMetadataAndUnlock(io(), commitStatus, newMetadataLocation, lock);
```

Comment:
```suggestion
      cleanupMetadataAndUnlock(io(), commitStatus, newMetadataLocation, lock);
```
I don't think you need to qualify this method

Reply:
It expects interface with static method . Plz let me know if you think other way around. 
<img width="1111" alt="Screenshot 2024-03-25 at 10 21 00 PM" src="https://github.com/apache/iceberg/assets/4146188/9d1f4f78-c607-41f2-9c10-04138c74a566">